sdk/cosmosdb/cosmos/README.md

```diff
@@ -3,14 +3,22 @@
 [![latest npm badge](https://img.shields.io/npm/v/%40azure%2Fcosmos/latest.svg)][npm]
```

Comment:
Are any changes here intended? I think you may just want to revert to the version of the file on master